uptane-whitepaper.md

```diff
@@ -1 +1,109 @@
+
+## **Starting conversations about the challenges ahead**
+As vehicles edge ever closer to the designs envisioned in novels and movies in the mid-20th century, there will be new questions to address. Currently, the Uptane group is opening conversation on two such issues. Look for new whitepapers at the end of 2021 and in early 2022 to deal with the following issues.
+
+*Allowing access to ECUs for emergency updates from federal/state/local governments*
+
+As SOTA delivery of updates becomes more sophisticated, government agencies and regulatory bodies, such as the U.S. Department of Transportation or its state or local equivalents, the Department of Homeland Security, or the Federal Emergency Management Agency,  may require automakers to grant them full control over a vehicle in emergency situations. These infrequent but important OTA broadcasts could be due to emergency or disaster routing, scheduled road work, or traffic conditions due to closed exits, failed stoplights, or other conditions.  Accommodating this access will require some re-thinking of how Uptane is configured, particularly in how to prioritize delegations, and perhaps to accommodate a dual Director set-up.
+
+*Security issues related to the use of aftermarket materials*
+
+Aftermarket companies refurbish and reuse equipment following end-of-life support from OEMs. This means introducing ECUs to a vehicle that the OEM has no control over. In addition, because aftermarket suppliers may not have access to the original design, they often reverse engineer the parts to figure out how it works. 
+Such an approach perhaps keeps these suppliers from being able to glean all relevant design information about the ECU.
+
+The white paper will address a number of issues, including the answers to the following questions:
+
+- How do we deal with aftermarket ECUs that do not have their own Primary? 
+- Can they leverage an OEM's Director repository?
+- If an aftermarket ECU does have its own Primary, is each capable of controlling a mutually exclusive set of ECUs?
+- Could ownership of the Director be delegated to a third party or owner?
+
+## **Learn More**
+The best place to learn more about Uptane is to go to its [website](https://uptane.github.io/participate.html). Here you can read more on the specification, review the current version of the [Uptane Standard for Design and Implementation](https://uptane.github.io/papers/uptane-standard.1.1.0.html) and the [Deployment Best Practices](https://uptane.github.io/papers/uptane-deployment-best-practices-1.1.0.html) volume, as well as conference presentations, testing information, and other data. We welcome questions, feedback, and suggestions on these materials, the website, or any other aspect of this project. Feel free to raise issues on GitHub or email feedback to jcappos@nyu.edu.
```

Comment:
We probably should have a "latest" link for the latest versions so we do not need to keep updating these links.

Reply:
This is a good point. Let me check with @tkfu who I believe did set up such a link. @tkfu can you verify this and, if so, provide that link?